mitodl/hugo-course-publisher
# Download course material button

#### Pre-Flight checklist


#### What are the relevant tickets?
https://trello.com/c/bVQwTyeU/170-download-course-material-should-be-a-button-in-course-info

#### What's this PR do?
Updates sample markdown with removed Download Course Materials sections and adds in a placeholder button in course_info for now.

#### How should this be manually tested?
Run the site, verify that the Course Download sections are removed and that the download button exists in Course Info.
